pkg/eve/runme.sh

```diff
@@ -67,10 +67,13 @@ This allows you to overwrite files in the EVE-OS config partition with
 your own local modifications of those files (must have the same name).
 
 Example:
-docker run -v $HOME/eve-overrides:/in --rm lfedge/eve:latest installer_raw > installer.raw
+docker run -v /path/to/eve/conf:/in --rm lfedge/eve:latest installer_raw > installer.raw
+or
+docker run -v /path/to/eve/conf/server:/in/server -v /path/to/eve/conf/soft_serial:/in/soft_serial --rm lfedge/eve:latest installer_raw > installer.raw
```

Comment:
soft_serial might be a bad example because I don't think it is in the installer image.
root-certificate.pem or authorized_keys would be better examples.

Reply:
but `soft_serial` is part of the reason for the change: we rely on it in line 143 to set IMAGE_UUID

Reply:
That's only for the live image and never for the installer image, which is your example.

And the installer image is the correct example for the users - only developers care about the live image

Reply:
Is there a live image use case where the uuidgen isn't sufficient and you want to specify a particular soft serial?

Reply:
I have the following wrokflow for my live images: I always keep the same `soft_serial` file in eve/conf that gets copied into the live image instead of generating a new UUID. This way I don't have to change my scripts for automatic deployment etc, because the test node always has the same ID across all versions and controllers.

However I see your point about EVE container images being mainly used to create installer images, so I change the doc according to your comment.